qiskit/circuit/quantumcircuit.py

```diff
@@ -196,6 +196,17 @@ def calibrations(self):
         """
         return dict(self._calibrations)
 
+    @calibrations.setter
+    def calibrations(self, cals_input):
+        """Set the circuit calibration data from a dictionary of calibration definition.
+
+        Args:
+            cals_input (dict): A dictionary of input in th format
+                {'gate_name': {(qubits, gate_params): schedule}}
+        """
+        self._calibrations.clear()
```

Comment:
Is this line necessary?

Reply:
I believe that the cals will be empty but left it there just in case. I can remove them though.

Reply:
Fixed in https://github.com/Qiskit/qiskit-terra/pull/5049/commits/7174ad30f9c0713a280e2ec346160430a76869d8